canvas/raster.go

```diff
@@ -4,15 +4,22 @@ import (
 	"image"
 	"image/color"
 	"image/draw"
+	"os"
+	"runtime/debug"
+	"time"
+
+	"github.com/steveoc64/memdebug"
```

Comment:
Any reason to leave the memdebug stuff in?  Seems like that was for debugging and shouldn't be in there for everyone else?

Reply:
Ha there was a race condition in my viewing of the review - looks like you removed the debugging while I was typing this up 😅 where do we put the mutex for this??

Reply:
lol - actually lolled when I read that.

my bad - I should self review _before_ posting the PR, not after :)